narwhal/types/build.rs

```diff
@@ -145,15 +145,6 @@ fn build_anemo_services(out_dir: &Path) {
                 .codec_path(codec_path)
                 .build(),
         )
-        .method(
```

Comment:
I think these methods are unused. For worker info, I'm not sure updating worker info within an epoch can / should be implemented, since worker addresses lives on chain right now. It should be easy to add back these methods if we decide to use it later?

Reply:
Yup good point. Alot has changed since then, we can re add it later